CRM/Contact/BAO/Query.php

```diff
@@ -4208,6 +4208,12 @@ public function relationship(&$values) {
       }
       $sql = "
         CREATE TEMPORARY TABLE {$relationshipTempTable}
+          (
+            `contact_id` int(10) unsigned NOT NULL DEFAULT '0',
+            `contact_id_alt` int(10) unsigned NOT NULL DEFAULT '0',
+            KEY `contact_id` (`contact_id`),
+            KEY `contact_id_alt` (`contact_id_alt`)
+          )
```

Comment:
@twomice don't you need to have a column for the civicrm_relationship_id as well?)

Reply:
I guess not. I mean, it's been working just fine without that column so far.